### ISSUE TITLE
Cannot ssh into a running pod/container -- rpc error: code = 2 desc = oci runtime error: exec failed: container_linux.go:247: starting container process caused "process_linux.go:110: decoding init error from pipe caused \"read parent: connection reset by peer\""  command terminated with exit code 126

### ISSUE THREAD

**Issue by sverma37** [openshift/origin#21590]
Cannot ssh into a running pod/container

##### Version
openshift v3.9.0+ba7faec-1
kubernetes v1.9.1+a0ce1bc657
etcd 3.2.16

##### Steps To Reproduce
1. Installed openshift 3.9 with one master and 2 nodes
2. Deployed Camunda image using openshift. The image got deployed and the service is running. However, I can not ssh into the pod

##### Current Result
oc rsh camunda-1-bk6h8

rpc error: code = 2 desc = oci runtime error: exec failed: container_linux.go:247: starting container process caused "process_linux.go:110: decoding init error from pipe caused \"read parent: connection reset by peer\""
command terminated with exit code 126

##### Expected Result
Logged in the container's shell

**Comment by AleksanderGrzybowski** [comment]
Hello, we have the same problem, all containers with shell probes are stuck in a crashloop because of that. Removing the probe looks like a workaround for now, but there is no way to run commands in containers.

**Comment by ephracis** [comment]
I am seeing similar issues today. I don't think it is related to OpenShift but to something else.

Steps to reproduce:

1. Create virtual machine with latest CentOS
2. `yum -y install docker`
3. `systemctl start docker`
4. `docker run -d --name test nginx`
5. `docker exec test id`

Some info:

```
$ cat /etc/redhat-release
CentOS Linux release 7.5.1804 (Core)
$ uname -a
Linux test.example.com 3.10.0-862.14.4.el7.x86_64 #1 SMP Wed Sep 26 15:12:11 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
$ docker info
Containers: 1
 Running: 1
 Paused: 0
 Stopped: 0
Images: 1
Server Version: 1.13.1
Storage Driver: overlay2
 Backing Filesystem: xfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: journald
Cgroup Driver: systemd
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: inactive
Runtimes: docker-runc runc
Default Runtime: docker-runc
Init Binary: /usr/libexec/docker/docker-init-current
containerd version:  (expected: aa8187dbd3b7ad67d8e5e3a15115d3eef43a7ed1)
runc version: 5eda6f6fd0c2884c2c8e78a6e7119e8d0ecedb77 (expected: 9df8b306d01f59d3a8029be411de015b7304dd8f)
init version: fec3683b971d9c3ef73f284f176672c44b448662 (expected: 949e6facb77383876aeff8a6944dde66b3089574)
Security Options:
 seccomp
  WARNING: You're not using the default seccomp profile
  Profile: /etc/docker/seccomp.json
 selinux
Kernel Version: 3.10.0-862.14.4.el7.x86_64
Operating System: CentOS Linux 7 (Core)
OSType: linux
Architecture: x86_64
Number of Docker Hooks: 3
CPUs: 1
Total Memory: 7.639 GiB
Name: test.example.com
ID: G6RY:7BY4:IBRN:B7RO:XU54:QBBN:YK4K:7EEL:OVVO:FYFV:UWIN:OJEV
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
Registries: docker.io (secure)
```

**Comment by LalatenduMohanty** [comment]
I also seen the same issue in CDK. With latest docker  we are seeing the issue, but with older docker `1.13.1-75` the issue is not there. 
```

$ sudo rpm -qa \| grep -i docker
--
docker-client-1.13.1-84.git07f3374.el7.x86_64
docker-1.13.1-84.git07f3374.el7.x86_64
docker-rhel-push-plugin-1.13.1-84.git07f3374.el7.x86_64
docker-common-1.13.1-84.git07f3374.el7.x86_64
```

**Comment by mrunalp** [comment]
@lsm5 @runcom Looks like we have a regression in build 84.

**Comment by AwesomeKai** [comment]
Experiencing the same 🤔 glad i'm not the only one

`version 1.13.1, build 07f3374/1.13.1 `

**Comment by rainmor** [comment]
> I've seen the same with a different container; In my case the fix was to downgrade docker from release 84 to 75.
> 
> `yum downgrade docker-1.13.1-75.git8633870.el7.centos.x86_64 docker-client-1.13.1-75.git8633870.el7.centos.x86_64 docker-common-1.13.1-75.git8633870.el7.centos.x86_64`

thanks for your solution。

**Comment by simongrzebieta** [comment]
New release of docker is at fault.
https://bugzilla.redhat.com/show_bug.cgi?id=1655214
https://access.redhat.com/solutions/3734981

**Comment by cassiussa** [comment]
The suggestion that @senax provided above worked for me as well.

```
yum downgrade docker-1.13.1-75.git8633870.el7.centos.x86_64 docker-client-1.13.1-75.git8633870.el7.centos.x86_64 docker-common-1.13.1-75.git8633870.el7.centos.x86_64
```

**Comment by mrunalp** [comment]
An updated fixed build is on the way.

**Comment by zhachaoy** [comment]
> I've seen the same with a different container; In my case the fix was to downgrade docker from release 84 to 75.
> 
> `yum downgrade docker-1.13.1-75.git8633870.el7.centos.x86_64 docker-client-1.13.1-75.git8633870.el7.centos.x86_64 docker-common-1.13.1-75.git8633870.el7.centos.x86_64`

`nice`

**Comment by yllierop** [comment]
I've seen this same issue as well. @mrunalp what's the status of a fix?

**Comment by simongrzebieta** [comment]
Red hat have said it's been fixed in version `docker-1.13.1-88`
> https://access.redhat.com/solutions/3734981
> This issue is resolved in docker-1.13.1-88 in https://access.redhat.com/errata/RHBA-2018:3796.

**Comment by rcsavage** [comment]
> yum downgrade docker-1.13.1-75.git8633870.el7.centos.x86_64 docker-client-1.13.1-75.git8633870.el7.centos.x86_64 docker-common-1.13.1-75.git8633870.el7.centos.x86_64

This EXACT issue is happening to me as well.   I had to downgrade docker the exact same way to overcome the OCI runtime error.

**Comment by rcsavage** [comment]
If RH says the ""fix"" is in docker RPM docker -1.13.1-88 that's great news, but it's not available in CentOS 7 yet.   This work around (i.e. downgrade) will have to be our option for now until the CentOS community delivers a new package.

**Comment by xsgordon** [comment]
> If RH says the ""fix"" is in docker RPM docker -1.13.1-88 that's great news, but it's not available in CentOS 7 yet. This work around (i.e. downgrade) will have to be our option for now until the CentOS community delivers a new package.

http://mirror.centos.org/centos/7/extras/x86_64/Packages/docker-1.13.1-88.git07f3374.el7.centos.x86_64.rpm

**Comment by AwesomeKai** [comment]
> > I've seen the same with a different container; In my case the fix was to downgrade docker from release 84 to 75.
> > `yum downgrade docker-1.13.1-75.git8633870.el7.centos.x86_64 docker-client-1.13.1-75.git8633870.el7.centos.x86_64 docker-common-1.13.1-75.git8633870.el7.centos.x86_64`
> 
> `nice`

> 👍 Great work around, works
